lib/board.rb

```diff
@@ -0,0 +1,69 @@
+class Board
+  attr_accessor :cells
+
+  def reset!
+    # can reset the state of the cells in the board
+    # sets the cells of the board to a 9 element array of " "
+    @cells = Array.new(9, " ")
+  end
+  def initialize
+    # sets the cells of the board to a 9 element array of " "
+    @cells = Array.new(9, " ")
+  end
+  def display
+    # prints the board
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+  def position(index)
+    # takes in user input and returns the value of the board cell
+    idx = input_to_index(index)
+    @cells[idx]
+  end
```

Comment:
It's good practice to use getter and setter methods to access instance and class variables when we have them.  In this case, we have `attr_accessor :cells` which gives us a getter and setter for the instance variable `@cells`.  In this case, and throughout the rest of the class, we can just reference the getter `#cells`, such as 
```
    idx = input_to_index(index)
    self.cells[idx]
```
or 
```
    idx = input_to_index(index)
    cells[idx]
```
The second way works since `self` is the implied receiver.